ui_framework/doc_site/src/views/expression/expression_example.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import {
+  GuideDemo,
+  GuidePage,
+  GuideSection,
+  GuideSectionTypes,
+  GuideText,
+} from '../../components';
+
+const expressionHtml = require('./expression.html');
+const expressionJs = require('raw!./expression.js');
+
+
+export default props => (
+  <GuidePage title={props.route.name}>
+    <GuideSection
+      title="Expression builder"
+      source={[{
+        type: GuideSectionTypes.HTML,
+        code: expressionHtml,
+      }]}
```

Comment:
Can you change this so that the JS is also surfaced in the source viewer?

```javascript
      source={[{
        type: GuideSectionTypes.HTML,
        code: expressionHtml,
      }, {
        type: GuideSectionTypes.JS,
        code: expressionJs,
      }]}
```